### PR TITLE
Removing windows and orphans.

### PR DESCRIPTION
**Pull request by dclunde**
Cal Poly Master's These Guidelines (2019) require all windows and orphans to be removed. This is done through adding infinite penalty to their occurrence.

### CODE REVIEW

**Comment by dclunde** [comment]
Hi Travis,

  I've added a second commit. This one deals with adding more committee members. Thanks for verifying the first one!

Dom

**Comment by AndrewGuenther** [comment]
Forgive me for just copy pasting a StackOverflow link, but would this approach work? It would be my preferred if possible: https://tex.stackexchange.com/a/30409

**Comment by dclunde** [comment]
@AndrewGuenther Yes, that is what I used in the commit shown above.

**Comment by AndrewGuenther** [comment]
I was referring to this solution:
```
\usepackage[all]{nowidow}
```

**Comment by dclunde** [comment]
@AndrewGuenther That's a good one! I implemented it in my commit (along with footnote spacing update)

File: main.tex [hunk 1]
```diff
@@ -28,6 +28,15 @@
 \setcounter{secnumdepth}{3}
 \setcounter{tocdepth}{3}
 
+% Added by Dominic Lunde (4/2019) to avoid windows and orphans
```
Comment (AndrewGuenther) [inline main.tex]: Not that I don't want to give you credit, but the author will appear in the diff, no reason to call out the author here in line.

File: main.tex [hunk 2]
```diff
@@ -28,6 +28,15 @@
...
+\usepackage[all]{nowidow}
+% Added by Dominic Lunde (4/2019) to fix spacing between footnote entries
```
Comment (AndrewGuenther) [inline main.tex]: Ditto the above

File: ucthesis.cls [hunk 3]
```diff
@@ -276,9 +276,15 @@
...
 \def\othermemberD#1{\gdef\@othermemberD{#1}}
+\def\othermemberE#1{\gdef\@othermemberE{#1}}
+\def\othermemberF#1{\gdef\@othermemberF{#1}}
+\def\othermemberG#1{\gdef\@othermemberG{#1}}
+\def\othermemberH#1{\gdef\@othermemberH{#1}}
+\def\othermemberI#1{\gdef\@othermemberI{#1}}
 
 % The names of your other committe members, one per line
-\def\othermembers#1{\gdef\@othermembers{#1}}
+% Commented out because not being used currently, could be investigated as different option at other time - DL 03/2019
```
Comment (AndrewGuenther) [inline ucthesis.cls]: No need to comment this out, just delete it if it isn't being used.

**Comment by dclunde** [comment]
Good comments! Thanks!

**Comment by AndrewGuenther** [comment]
I have one final ask: Could you take your last two commits and rebase them into their relevant changes? It will keep a cleaner git history that way. Once that's done, I'll go ahead and merge.

**Comment by dclunde** [comment]
I screwed it all up. It was my first time rebasing. I'm going to close this and just make a new pull request

**Pull request closed without merging**